### PR TITLE
Migrate class to attr

### LINKED ISSUE #11642: Change "unmuted" CSS class on amp-story to "muted" attribute
This reads slightly better, and matches the behavior and semantics of HTML media elements.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #11642 

- Shifts `unmuted` from class to attribute

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

**Comment by newmuis** [comment]
I think it would be great if we could also negate this and make it `muted` instead of `unmuted`, and automatically add the `muted` attribute when the component is built.

**Comment by prateekbh** [comment]
Done with the changes

File: extensions/amp-story/0.1/amp-story.js [hunk 1]
```diff
@@ -79,6 +79,16 @@ const FULLSCREEN_THRESHOLD = 1024;
...
 
+/**
+ * @private @const {string}
+ */
+const AUDIO_MUTED_ATTRIBUTE = 'muted';
+
+/**
+ * @private @const {string}
+ */
+const AUDIO_UNMUTED_ATTRIBUTE = 'unmuted';
```
Comment (choumx) [inline extensions/amp-story/0.1/amp-story.js]: We should only need a single boolean attribute. Jon suggested `muted`.

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: My bad, misread the description. 
Done now

**Pull request merged**